packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -36,7 +36,8 @@ dependencies: {{#enable_dev_menu}}
   jaguar_jwt: ^3.0.0 # This package is used for the local server (if not used, can be removed)
   json_annotation: ^4.9.0
   jwt_decoder: ^2.0.1{{#enable_pin_code}}
-  local_session_timeout: ^3.2.0{{/enable_pin_code}}
+  local_session_timeout: ^3.2.0{{/enable_pin_code}}{{#enable_feature_onboarding}}
+  open_mail: ^0.0.5{{/enable_feature_onboarding}}
```

Comment:
Can we use more established packaged other than `open_mail` ?

Reply:
At first I tried to use the more established `open_mail_app` package, which we've used before, however it seems to have issues compiling an Android build with recent updates, given that the package hasn't been updated in 2 years:
<img width="750" alt="image" src="https://github.com/user-attachments/assets/935ca54a-7abd-41d5-87bc-ba78d12bd1c6" />

Given the other available options, I went with the `open_mail`, which seems to be mostly the same package (with the same API), just updated:
<img width="705" alt="image" src="https://github.com/user-attachments/assets/1d1e2f5b-744e-43b3-9dab-0ef2d6d96f4a" />

Another option would be to implement our own solution for a mail app picker, but this would take more time

Reply:
As discussed, I've removed the `open_mail` dependency and replaced it with `url_launcher` opening a `mailto:` link